Developers/Writing_Linter_Bears.rst

```diff
@@ -387,5 +387,5 @@ Congratulations!
 Where to Find More...
 ---------------------
 
-If you need more information about the ``@linter`` decorator, refer to the API
-documentation.
+If you need more information about the ``@linter`` decorator, refer to the `API
+documentation. <http://docs.coala.io/en/stable/Users/Tutorials/Linter_Bears.html>`_
```

Comment:
Yes, I didn't know that it is not the complete API documentation.
Ok, correcting the commit.